src/App.test.js

```diff
@@ -70,6 +69,7 @@ describe('Signing In', () => {
     await page.type('.password', person.password)
     await page.click('.remember')
     await page.click('.signIn')
+    await page.waitFor(".Navigation")
```

Comment:
Err, try waiting for a set time, because if this happens to redirect to something aside from the Dashboard, where there is no Navigation-class element, this would stall (as would be the case currently on stage).